app/src/main/java/com/iot/mobiledevelopment/DetailActivity.java

```diff
@@ -0,0 +1,40 @@
+package com.iot.mobiledevelopment;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.squareup.picasso.Picasso;
+
+import static com.iot.mobiledevelopment.MoviesFragment.EXTRA_DESCRIPTION;
+import static com.iot.mobiledevelopment.MoviesFragment.EXTRA_TITTLE;
+import static com.iot.mobiledevelopment.MoviesFragment.EXTRA_URL;
+import static com.iot.mobiledevelopment.MoviesFragment.EXTRA_YEAR;
```

Comment:
I suppose constants can be moved to the separate class (if you have inspiration to do this =))